core/test_network/src/lib.rs

```diff
@@ -847,11 +847,13 @@ impl TestConfiguration for Configuration {
         let (public_key, private_key) = KeyPair::generate().unwrap().into();
         configuration.public_key = public_key;
         configuration.private_key = private_key;
+        configuration.sumeragi.connect_peers_period_seconds = 2; // Apply this to the default, non test configuration after #2875 is completed.
         configuration
     }
 
     fn pipeline_time() -> Duration {
         Duration::from_millis(Self::test().sumeragi.pipeline_time_ms())
+            + Duration::from_secs(Self::test().sumeragi.connect_peers_period_seconds.into())
```

Comment:
We need to somehow reduce the load on the actor system. I don't know if we should add the `connect_peers_period_seconds` in this **particular** case, but given that the `pipeline_time()` is used as a shorthand for "time when the peers would be active again", this makes sense. 

I suggest renaming the function to `recovery_time`, to avoid further confusion.